pkg/cloudprovider/aws/aws.go

```diff
@@ -34,87 +34,62 @@ import (
 	"github.com/golang/glog"
 )
 
-// Abstraction over EC2, to allow mocking/other implementations
-type EC2 interface {
-	// Query EC2 for instances matching the filter
-	Instances(instIds []string, filter *ec2InstanceFilter) (resp *ec2.InstancesResp, err error)
-}
+/****************************************************************************************************
```

Comment:
We generally don't use /* */ comments in Kubernetes code, as it's often a pain during debugging. Please change to // here and everywhere below.